templates/scripts/create_bootstrap_node_evm_compose_file.sh

```diff
@@ -192,7 +192,19 @@ if [ "${enable_domains}" == "true" ]; then
       echo "      \"--reserved-nodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
       echo "      \"--bootstrap-nodes\", \"${addr}\"," >> ~/subspace/docker-compose.yml
     done
-
+    # auto domain
+      echo '      "--",'
```

Comment:
This is not correct unfotunately

We would need to create two compose file. One for EVM and one for AutoID
Our node currently does not support running multiple domains in the single binary